### PR TITLE
Blocked list inaccessiable on smaller screens

### PR DESCRIPTION
**Pull request by yougotwill**
## Before

Screenshot 2024-07-26 at 15 05 53
Screenshot 2024-07-26 at 15 05 18

## After

Screenshot 2024-07-26 at 15 09 03
Screenshot 2024-07-26 at 15 08 55

### CODE REVIEW

File: ts/components/MemberListItem.tsx [hunk 1]
```diff
@@ -114,7 +118,7 @@ export const MemberListItem = (props: {
     >
       <StyledInfo>
         <AvatarItem memberPubkey={pubkey} isAdmin={isAdmin || false} />
-        <StyledName>{memberName}</StyledName>
+        <StyledName maxNameWidth={maxNameWidth}>{memberName}</StyledName>
```
Comment (Bilb) [inline ts/components/MemberListItem.tsx:121]: should probably be renamed to maxWidth once we are targetting the StyledName element.
I.e. keep   `maxNameWidth?: string;` on line 79, but change line 121 to 
`        <StyledName maxWidth={maxWidth}>{memberName}</StyledName>`

File: ts/components/settings/BlockedList.tsx
Comment (Bilb) [inline ts/components/settings/BlockedList.tsx:1]: a lot of UI changes in this file. I hope it doesn't break one theme, somehow

File: ts/components/settings/BlockedList.tsx [hunk 2]
```diff
@@ -78,6 +63,7 @@ const BlockedEntries = (props: {
               onSelect={addToSelected}
               onUnselect={removeFromSelected}
               disableBg={true}
+              maxNameWidth={'33vw'}
```
Comment (Bilb) [inline ts/components/settings/BlockedList.tsx:66]: I'd rather not have this. Can we not fix it with a flex-shrink on the StyledName element and a maxWidth on the row itself?

**Review by Bilb — CHANGES_REQUESTED** [review]

File: ts/components/MemberListItem.tsx (hunk 1 above)
Comment (yougotwill) [reply ts/components/MemberListItem.tsx]: Fixed

File: ts/components/settings/BlockedList.tsx
Comment (yougotwill) [reply ts/components/settings/BlockedList.tsx:1]: Trust me bro the styling mostly comes from onboarding from a few months ago. I was very thorough 🙏

File: ts/components/settings/BlockedList.tsx (hunk 2 above)
Comment (yougotwill) [reply ts/components/settings/BlockedList.tsx:66]: Part of the issue in this case is that we still want the radio buttons to be at the end of the container. If we limit the width of the entire thing it looks weird 😅

**Review by yougotwill — COMMENTED** [review]
Responded

File: ts/components/settings/BlockedList.tsx (hunk 2 above)
Comment (yougotwill) [reply ts/components/settings/BlockedList.tsx:66]: Screenshot 2024-07-26 at 16 07 22

Unless I'm missing something?

Comment (Bilb) [reply ts/components/settings/BlockedList.tsx:66]: idk, I'd rather fix this properly :eyes:

Comment (yougotwill) [reply ts/components/settings/BlockedList.tsx:66]: @Bilb Let's discuss further

File: ts/components/MemberListItem.tsx [hunk 3]
```diff
@@ -53,13 +53,15 @@ const StyledInfo = styled.div`
...
-const StyledName = styled.span`
+const StyledName = styled.span<{ maxWidth?: string }>`
   font-weight: bold;
   margin-inline-start: var(--margins-md);
   margin-inline-end: var(--margins-md);
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
+
+  ${props => props.maxWidth && `max-width: ${props.maxWidth};`}
```
Comment (yougotwill) [inline ts/components/MemberListItem.tsx:64]: I guess we don't need to keep this anymore right?

**Review by Bilb — APPROVED** [review]

File: ts/components/settings/BlockedList.tsx (hunk 2 above)
Comment (yougotwill) [reply ts/components/settings/BlockedList.tsx]: Discussed and resolved

**Pull request merged**